www/include/config.php

```diff
@@ -39,9 +39,9 @@
 		),
 	);
 
-	$GLOBALS['cfg']['wof_data_dir'] = '/usr/local/mapzen/whosonfirst-data-venue-us-new-york/data';
+	$GLOBALS['cfg']['wof_data_dir'] = '/usr/local/mapzen/whosonfirst-data';
 	$GLOBALS['cfg']['wof_github_owner'] = 'whosonfirst-data';
-	$GLOBALS['cfg']['wof_github_repo'] = 'whosonfirst-data-venue-us-new-york';
+	$GLOBALS['cfg']['wof_github_repo'] = 'whosonfirst-data-venue-us-california';
```

Comment:
I already mentioned this via Slack, but this should probably not be in the main `config.php`.